jstests/core/explain_verbosity_unknown.js

```diff
@@ -0,0 +1,22 @@
+//Test whether unknown commands are accepted in explain function
+
+(function() {
+	
+	"use strict";
+	
+	var unknownCollection = db["jstestDB"];
+
+	//insure the collection has an item
+	unknownCollection.insert({a:1});
+
+	//check with finding item that exist
+	assert.commandFailed(db.runCommand({explain: {find: 'a'}, verbosity: 'queryPlanner', unknown: 1}), "unknown argument command extra");
+	assert.commandFailed(db.runCommand({explain: {find: 'a'}, unknownArg: 'queryPlanner'}), "unknown verbosity command");
```

Comment:
Each of these are executing a query of the collection named 'a'. I think you meant to execute them on the collection mentioned on line 7 above, so you should use `collection.getName()` instead of `'a'`.